lib/aruba/cucumber/command.rb

```diff
@@ -136,7 +136,30 @@
   expect(last_command_started.output).to have_output_size size.to_i
 end
 
-Then(/^(?:the )?(output|stderr|stdout)(?: from "([^"]*)")? should( not)? contain( exactly)? "([^"]*)"$/) do |channel, cmd, negated, exactly, expected|
+Then(/^(?:the )?(output|stderr|stdout) should( not)? contain( exactly)? "([^"]*)"$/) do |channel, negated, exactly, expected|
```

Comment:
Whilst this is extremely mutable, actually BDD advocates against this.

It would be better to have twice as many cukes with half the permeability or a quarter e.t.c.

I'm happy for you to defer this, but I'm against introducing more of it

Reply:
I agree but I'm going to defer this to the 1.0 or post-1.0 release.

Reply:
Actually, maybe I should clean that up now.

Reply:
Up to you, maybe make an issue for tidying up. It's a big refactor task though. So perhaps one best done another time with a clear head/coffee

Reply:
Ok, just to make sure I understand correctly, you're saying this should be split so that there are several definitions reading something like:

```ruby
Then(/^the output should contain "([^"]*)"$/ do |expected|
# ...
Then(/^the stderr should contain "([^"]*)"$/ do |expected|
# ...
# etc.
```

Right?

Reply:
1000%, I'm not saying you need to split on each option. But just removing one permutation would make it so much easier to read / review.

Sometimes unDRYing it makes it so much easier to follow. This stuff is quite tricky.

Reply:
I created #640 for this.